demo/core_13_$.utils.code.js

```diff
@@ -105,3 +105,44 @@ $.utils.code.valueToSourceSafe = function(value) {
     throw e;
   }
 };
+
+
+$.utils.code.getTempObj = function(id) {
```

Comment:
Make it its own object; method should be called something like `$.<whatever>.tempRefs.get` or `$.<whatever>.refDB.get` or ??

Also consider if there should be a constructor for DB instances.

Reply:
Changed to $.db.tempObject.getObjById